### PR TITLE
fix: add enum support to twilio-node

### PR DESCRIPTION
**Pull request by childish-sambino**

### CODE REVIEW

File: examples/node/lib/rest/api/v2010/account.ts [hunk 1]
```diff
@@ -20,14 +20,34 @@ const deserialize = require("../../../base/deserialize");
 const serialize = require("../../../base/serialize");
 import { CallListInstance } from "./account/call";
 
+type TestStatus =
```
Comment (beebzz) [inline examples/node/lib/rest/api/v2010/account.ts:23]: Changes look good, but why aren't we declaring enums the [traditional way](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)?

File: scripts/generate.sh [hunk 2]
```diff
@@ -44,5 +44,5 @@ OUT_DIR=examples/java/src/main/java/com/twilio/rest
 generate twilio-java
 
 OUT_DIR=examples/node/lib/rest
-generate twilio-node
+generate twilio-node --global-property skipFormModel=false
```
Comment (beebzz) [inline scripts/generate.sh:47]: Isn't this already being set [here](https://github.com/twilio/twilio-oai-generator/pull/281/files#diff-53db1b6599656db8b4a898bdcc6632d6bbf39605ade0cb914f8f29c840927431R49)?

Comment (childish-sambino) [reply scripts/generate.sh:47]: This script is for building the examples. The other is for building the actual helpers.

File: examples/node/lib/rest/api/v2010/account.ts (hunk 1 above)
Comment (childish-sambino) [reply examples/node/lib/rest/api/v2010/account.ts:23]: Matching existing functionality.

**Review by beebzz — APPROVED** [review]

**Pull request merged**